aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/UpdateHandler.java

```diff
@@ -112,15 +117,33 @@ public UpdateHandler(TagHelper tagHelper) {
             logger.log(String.format("INFO Updated tags for patch baseline %s %n", baselineId));
 
             // Set to default patch baseline
-            if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+           // if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+            if(model.getDefaultBaseline()!=null && BooleanUtils.isTrue(model.getDefaultBaseline()) ){
+
                 RegisterDefaultPatchBaselineRequest registerDefaultPatchBaselineRequest = RegisterDefaultPatchBaselineRequest.builder()
                         .baselineId(baselineId)
                         .build();
                 RegisterDefaultPatchBaselineResponse registerDefaultPatchBaselineResponse =
                         proxy.injectCredentialsAndInvokeV2(registerDefaultPatchBaselineRequest, ssmClient::registerDefaultPatchBaseline);
 
-                logger.log(String.format("INFO Registered patch baseline %s to default patch baseline successfully %n", baselineId));
+                if(registerDefaultPatchBaselineResponse== null){
```

Comment:
does this mean it's successful?

Reply:
ii don't remember what the successful response should be

Reply:
Now updated code changes are added to the file